modules/tensor_mechanics/src/materials/RecomputeRadialReturn.C

```diff
@@ -140,6 +140,18 @@ RecomputeRadialReturn::getIsotropicShearModulus()
   return shear_modulus;
 }
 
+Real
+RecomputeRadialReturn::getIsotropicBulkModulus()
+{
+  Real shear_modulus = getIsotropicShearModulus();
+  Real dil_modulus = _elasticity_tensor[_qp](0,0,0,0);
```

Comment:
Would you be willing to use the full name of the modulus here?  I'm guessing that `dil_modulus` refers to dilatation or volumetric?  The full name will make it easier for new users to google the term and hopefully harder for continuing users to forget the name when we go back to add in the documentation in the new documentation system.